src/steamship/agents/schema/text_splitters.py

```diff
@@ -10,22 +10,22 @@ def split_text(self, text: str) -> List[str]:
         """Split the incoming text into strings"""
         raise NotImplementedError()
 
-    def chunk_text_to_tags(self, text: str, kind: str, name: str = None) -> List[Tag]:
+    def chunk_text_to_tags(self, block: Block, kind: str, name: str = None) -> List[Tag]:
         """Split the incoming text into strings, and then wrap those strings in Tags"""
-        if text is not None and text != "":
-            text_splits = self.split_text(text)
+        if block.text is not None and block.text != "":
```

Comment:
do you need to check `is_text()` at all?

Reply:
👍